client/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { LineChart, Line, CartesianGrid, XAxis, YAxis, Tooltip } from 'recharts';
+import './App.css';
+
+const URL = 'http://localhost:5000/data'
```

Comment:
I'd just use `/data` and use the `devServer` config in `webpack`. 

(just FYI)